Source/RomanNumeral.c

```diff
@@ -10,7 +10,144 @@ static char * Append(char *output, const char *toAppend)
 {
    return output + sprintf(output, "%s", toAppend);
 }
-
 void RomanNumeral_Convert(unsigned input, char *output)
 {
+   int i, hundreds, tens, units;
+   i = 0;
+   hundreds = (input / 100) % 10;
+   tens = (input - (hundreds*100)) / 10;
+   units = (input - (hundreds*100 + tens*10));
+   for(i = 0; i <=100; i++)
+      {
+         output[i] = '\0';
+      }
+   switch (hundreds)
+      {
+      case 1: 
+         *output = 'C';
+         output++;
+         break;
+      default: 
+         break;
+      }
+
+   switch (tens)
```

Comment:
looks like you are using case by case - is there any way you can simplify this by using loops? 

Looks like anytime you need to cover a bigger range, you will need to add a bunch of lines. Pretty sure you can find something simpler

Reply:
Ok, I will work on it.